src/Components/TodoList.jsx

```diff
@@ -15,16 +16,12 @@ const toDoList = [
     },
 ]
 
-export function TodoList(){
+export function TodoList() {
     return (
-        <>
-            <ul>
-                {toDoList.map(function (item) {
-                    return <li key={item.id}>{item.title}</li>
-                })}
-            </ul>
-        </>
-
-
+        <ul>
+            {toDoList.map (function (item) {
+                return <TodoListItem key={item.id.toString()} todo={item}/>;
```

Comment:
the key doesn't typically need to be a string. This'll work, just adding that the id property should work on its own